app/logic/courseManagement.py

```diff
@@ -70,32 +70,39 @@ def createCourse(courseData, instructorsDict):
         )
     for instructor in instructorsDict["instructors"]:
         CourseInstructor.create(course=course, user=instructor.username)
+    return course
 
 def updateCourse(courseData, instructorsDict):
     """
         This function will take in courseData for the SLC proposal page and a dictionary
         of instuctors assigned to the course and update the information in the db.
     """
-
-    for toggler in ["regularOccurenceToggle", "slSectionsToggle", "permanentDesignation"]:
-        courseData.setdefault(toggler, "off")
-    status = CourseStatus.get(CourseStatus.status == "Pending")
-    course = Course.update(
-        courseName=courseData["courseName"],
-        courseAbbreviation=courseData["courseAbbreviation"],
-        courseCredit=courseData["credit"],
-        isRegularlyOccuring=("on" in courseData["regularOccurenceToggle"]),
-        term=courseData['term'],
-        status=status,
-        isAllSectionsServiceLearning=("on" in courseData["slSectionsToggle"]),
-        serviceLearningDesignatedSections=courseData["slDesignation"],
-        isPermanentlyDesignated=("on" in courseData["permanentDesignation"]),
-    ).where(Course.id == courseData['courseID']).execute()
-    for i in range(1, 7):
-        (CourseQuestion.update(questionContent=courseData[f"{i}"])
-                    .where((CourseQuestion.questionNumber == i) & (CourseQuestion.course==courseData["courseID"])).execute())
-    removeInstructors = CourseInstructor.delete().where(CourseInstructor.course == courseData["courseID"]).execute()
-    for instructor in instructorsDict["instructors"]:
-        if not CourseInstructor.select().where(CourseInstructor.course==courseData["courseID"], CourseInstructor.user==instructor).exists():
-            addInstructors = CourseInstructor.create(course=courseData["courseID"], user=instructor)
+    try:
+        coursebyid= Course.get_by_id(courseData['courseID'])
```

Comment:
I would just call this one `course`, and then for confusion just remove the existing `course` variable from the `.update()`. It wasn't being used, anyway.